reference/sectionC_interpretation.rst

```diff
@@ -23,18 +29,20 @@ Interpretation of imports
 Units reference
 ---------------
 
-#. A units reference SHALL be a CellML identifier and SHALL be interpreted dependent on the context of the :ref:`CellML model<specA_cellml_model>` in which it occurs, according to the units referencing rules defined later in this section.
-
-#. A :ref:`CellML infoset<specA_cellml_infoset>` MUST NOT contain a units reference for which no referencing rule can be held to have been followed.
+#. A units reference SHALL be a CellML identifier and SHALL be  interpreted dependent on the context of the :ref:`CellML model<specA_cellml_model>` in which it occurs.
 
 #. The units referencing rules are:
 
-   #. Where there is a :code:`units` element with a :code:`name` attribute identical to the units reference, then the units reference SHALL refer to that :code:`units` element.
+   #. Where there is a :code:`units` element with a :code:`name` attribute identical to the units reference within the same infoset, then the units reference SHALL refer to that :code:`units` element.  
 
    #. Where there is an :code:`import units` element in the :ref:`CellML infoset<specA_cellml_infoset>`, such that the :code:`import units` element has a :code:`name` attribute identical to the units reference, then the units reference SHALL be treated with respect to referencing rules as if the units reference appeared in the imported infoset, and referring to the :code:`name` specified in the :code:`units_ref` attribute of the :code:`import units` element.
 
    #. Where the units reference is equal to the value in the ‘Name’ column of the :ref:`Built-in units table<table_built_in_units>`, then the units reference SHALL be a reference to the built-in units corresponding to that row of the table.
 
+#. A :ref:`CellML infoset<specA_cellml_infoset>` MUST NOT contain a units reference for which no referencing rule can be held to have been followed.
```

Comment:
This needs to change (or to go?).
It's left over from where we had a list of criteria to determine which rule was "held to be followed", and then ended saying at least one should be "held to be followed". We reworded all of that (phew!), so this one should change too

Reply:
The list of criteria for units referencing are above still?